library/spdm_crypt_lib/fips/libspdm_selftest_aes_gcm.c

```diff
@@ -0,0 +1,145 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_crypt_lib.h"
+
+#if LIBSPDM_FIPS_MODE
+
+/**
+ * AES_GCM self_test
+ **/
+bool libspdm_fips_selftest_aes_gcm(void)
+{
+    bool result;
+    result = true;
+
+#if LIBSPDM_AEAD_GCM_SUPPORT
+    uint8_t output_ciphertext[1024];
+    uint8_t output_plaintext[1024];
+    uint8_t output_tag[1024];
+    size_t output_ciphertext_size;
+    size_t output_plaintext_size;
+
+    output_ciphertext_size = sizeof(output_ciphertext);
+    output_plaintext_size = sizeof(output_plaintext);
+
+    /*Key to use crypto operation*/
+    const uint8_t key[16] = {
```

Comment:
You can get rid of the explicit array sizes.

Reply:
Yes. I deleted all explicit array sizes.